packages/ARU/PKGBUILD

```diff
@@ -0,0 +1,21 @@
+pkgname=kapitan
+pkgver=0.27.0
+pkgrel=1
+pkgdesc="Generic templated configuration management for Kubernetes, Terraform and other things"
+arch=('any')
+license=('Apache')
+url="https://kapitan.dev/"
+depends=('python' 'python-setuptools')
+provides=("kapitan")
+conflicts=("kapitan")
+source=("https://github.com/deepmind/kapitan/archive/v${pkgver}.tar.gz")
+md5sums=('3840e3501581c68a09055288ba17d9b9')
```

Comment:
Is this the md5 of the tarball? If so, can we generate programmatically?

Reply:
Yeah I was thinking of updating the CI pipeline to automatically update this file and change the version and the calculate the md5 hash after every release push the change to ARU so we don't have to worry about manually updating this after every release

Reply:
Yeah that'd be cool

Reply:
So about the question, I asked above should I update Kapitan CI to include reclass in the release archives or Update PKGBUILD to clone reclass and inject into Kapitan before compiling?

Reply:
Including reclass is fine as it's already downloaded and pinned

Reply:
ok, I will update the Travis config and build scripts to reflect that. BTW is it a good idea to keep API keys like this ([.travis.yml#L56](https://github.com/deepmind/kapitan/blob/1eee6a623bd2b0599046837f6020fe4f680b93bb/.travis.yml#L56)) inside the git repo in plain text as this public repo?. Is this intentional or someone accidentally check it into version control without realizing it?

Reply:
They are encrypted

Reply:
Ah yeah I just realized it and came to edit commit 😅

Reply:
> Including reclass is fine as it's already downloaded and pinned

Looks like I have to pull the reclass during build time (https://github.community/t5/How-to-use-Git-and-GitHub/Create-Release-that-contains-submodule/td-p/12073)